pandas/tests/extension/test_text.py

```diff
@@ -0,0 +1,111 @@
+import string
+
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas.core.arrays.text import TextArray, TextDtype
+from pandas.tests.extension import base
+
+
+@pytest.fixture
+def dtype():
+    return TextDtype()
+
+
+@pytest.fixture
+def data():
+    strings = np.random.choice(list(string.ascii_letters), size=100)
+    while strings[0] == strings[1]:
+        strings = np.random.choice(list(string.ascii_letters), size=100)
+
+    return TextArray._from_sequence(strings)
```

Comment:
Not too important, but for such things should we rather use `pd.array(.., dtype="text")` internally?

Reply:
Hmm probably. Unless we're worried about a bug in `pd.array()` perhaps throwing off tests? At least with this it's clear you're getting a TextArray?